### PR TITLE
[INTERNAL] JSModuleAnalyzer: Remove invalid log usage

### PR DESCRIPTION
**Pull request by tobiasso85**
Change usage from non existing log.debug to log.verbose.

This code was not yet covered by tests. The new tests ensure that the code is run through and no error is thrown.

### CODE REVIEW

**Comment by tobiasso85** [comment]
> I would rather fix the existing test. The log lines ARE already covered by the EVOBundle test in [JSModuleAnalyzer](https://github.com/SAP/ui5-builder/blob/master/test/lib/lbt/analyzer/JSModuleAnalyzer.js#L292-L320). But the test unfortunately totally stifles all errors.
> 
> The issue is caused by line https://github.com/SAP/ui5-builder/blob/master/test/lib/lbt/analyzer/JSModuleAnalyzer.js#L97,

Good point! Addressed it with 4ce5d9d

File: test/lib/lbt/analyzer/JSModuleAnalyzer.js [hunk 1]
```diff
@@ -287,8 +287,6 @@ test.cb("OldStyleBundleV2", (t) => {
 	);
 });
 
-// Note: this test still fails as the evo-style bundles don't declare the names of the
-// contained raw-modules any longer.
```
Comment (RandomByte) [inline test/lib/lbt/analyzer/JSModuleAnalyzer.js]: Was this comment removed intentionally?

Comment (codeworrior) [reply test/lib/lbt/analyzer/JSModuleAnalyzer.js]: I guess, yes. Because this TODO was, what #341 wanted to addressed (migration of 3562114).

Comment (tobiasso85) [reply test/lib/lbt/analyzer/JSModuleAnalyzer.js]: I thought this fix made the tests not fail anymore therefore I removed it

**Pull request merged**